nifi-nar-bundles/nifi-azure-bundle/nifi-azure-processors/src/main/java/org/apache/nifi/processors/azure/storage/utils/BlobAttributes.java

```diff
@@ -45,4 +45,6 @@ public final class BlobAttributes {
     public static final String ATTR_NAME_LENGTH = "azure.length";
     public static final String ATTR_DESCRIPTION_LENGTH = "Length of the blob";
 
+    public static final String ATTR_NAME_ERROR = "azure.error";
```

Comment:
Recommend changing this to `azure.error.code` for more precision, since there could be other error attributes.
```suggestion
    public static final String ATTR_NAME_ERROR_CODE = "azure.error.code";
```

Reply:
Agreed – and fixed.